test/doc/content/source/userobjects/InterfaceUO.md

```diff
@@ -0,0 +1,13 @@
+# InterfaceUO
+
+## IMPORTANT NOTE
+*** Any derived class of InterfaceUserObject does not support yet getMaterialProperty, please use auxvariables***
+
+## Description
+!syntax description /UserObjects/InterfaceUO
+
+InterfaceUO is a user object computing an average value across an interface. The kind of average value are the one available in  [InterfaceAverageTools](/InterfaceAverageTools.md).
```

Comment:
The kind of average value are the one available in -> The **various types** of **averages** are **described** in...